rpm/replace-openssl/Makefile.certificate

```diff
@@ -0,0 +1,82 @@
+UTF8 := $(shell locale -c LC_CTYPE -k | grep -q charmap.*UTF-8 && echo -utf8)
```

Comment:
I think it's a good idea to call the new directory rpm/shangmi, instead of replace-openssl;

And please move rpm/READMED.md to rpm/shangmi/README.md, and just introduce shangmi spec and other files.

How about you?

Reply:
> I think it's a good idea to call the new directory rpm/shangmi, instead of replace-openssl;
> 
> And please move rpm/READMED.md to rpm/shangmi/README.md, and just introduce shangmi spec and other files.
> 
> How about you?

OK, LGTM. I'll fix it.

Reply:
> I think it's a good idea to call the new directory rpm/shangmi, instead of replace-openssl;
> 
> And please move rpm/READMED.md to rpm/shangmi/README.md, and just introduce shangmi spec and other files.
> 
> How about you?

I fix it and re-push these patches now. Thanks@dongbeiouba @InfoHunter